### PR TITLE
CI: Remove functional tests with hdfs

### PR DESCRIPTION
**Pull request by maxknv**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [x]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit 09aa96370edaa5795144aa4548d343542fb03d6d with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/72531/09aa96370edaa5795144aa4548d343542fb03d6d/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/builds/report.html">✅ success</a></td></tr>
<tr><td>ClickBench</td><td>Runs [ClickBench](https://github.com/ClickHouse/ClickBench/) with instant-attach table</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/clickbench__aarch64_.html">✅ success</a></td></tr>
<tr><td>Compatibility check</td><td>Checks that <b>clickhouse</b> binary runs on distributions with old libc versions. If it fails, ask a maintainer for help</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/compatibility_check__aarch64_.html">✅ success</a></td></tr>
<tr><td>Docker keeper image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/docker_keeper_image.html">✅ success</a></td></tr>
<tr><td>Docker server image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/docker_server_image.html">✅ success</a></td></tr>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/72520/b6725d33d115bb3b5992388b6c67600389ae0843/docs_check.html">✅ success</a></td></tr>
<tr><td>Install packages</td><td>Checks that the built packages are installable in a clear environment</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/install_packages__aarch64_.html">✅ success</a></td></tr>
<tr><td>Integration tests</td><td>The integration tests report. In parenthesis the package type is given, and in square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/integration_tests__aarch64__[1_6].html">✅ success</a></td></tr>
<tr><td>Performance Comparison</td><td>Measure changes in query performance. The performance test report is described in detail <a href="https://github.com/ClickHouse/ClickHouse/tree/master/docker/test/performance-comparison#how-to-read-the-report">here</a>. In square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/performance_comparison__release__[1_4]/report.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/71b651d4a8f064f00f588a8387b14ad8d9da4055/unit_tests__asan_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh [hunk 1]
```diff
@@ -16,5 +16,5 @@ for _ in {1..1000}; do
     if [[ $elapsed -gt 30 ]]; then
         break
     fi
-done 2>&1 | grep -o -P 'Query memory limit exceeded' | sed -r -e 's/(.*):([a-Z ]*)([mM]emory limit exceeded)(.*)/\2\3/' | uniq
```
Comment (antaljanosbenjamin) [inline tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh:19]: Why did you change this? It doesn't seem to be connected with HDFS.

Comment (maxknv) [reply tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh:19]: it's not connected, yes. I just have an error seems related to different sed version:
`sed: -e expression #1, char 52: Invalid range end` it does not like [a-Z ]*
And I started to wonder what is the point of this sed|uniq commands in the pipe? Aren't they pointless?

Comment (maxknv) [reply tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh:19]: thanks for the review btw,
I planned to ask someone about this line;)

Comment (antaljanosbenjamin) [reply tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh:19]: > Aren't they pointless?

At this current form it is pointless. If you check the history of the file, you can see that it was not pointless. I will approve.

Goede gedaan, meneer!

**Review by antaljanosbenjamin — APPROVED** [review]

File: tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh (hunk 1 above)
Comment (maxknv) [reply tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh:19]: Köszi, haver!
